packages/spec/package.json

```diff
@@ -41,6 +41,7 @@
     "esbuild-loader": "^2.10.0",
     "jest-environment-hops": "15.0.0-nightly.1",
     "raw-body": "^2.4.1",
+    "react16.8": "npm:react@16.8.0",
```

Comment:
Why do we need to specifically request react@16.8? Shouldn't the react that is already available be sufficient?

Reply:
React 16.8 is part of our supported range. But I can as well remove the specific test for it.

Reply:
@ZauberNerd How do we want to go on with the additional test for React 16.8?

Reply:
I'd say we can remove that, since we have react@17 in our repository anyway.

Reply:
@ZauberNerd PTAL again